Lower stkBPT Emissions (#685)
bgd-labs/aave-proposals-v3: src/20250423_AaveV3Ethereum_LowerStkABPTEmissions/LowerStkABPTEmissions.md

![Screenshot 2025-04-03 at 21.55.51](https://hackmd.io/_uploads/SJ8k8uhaJg.png)
Source: https://aave.tokenlogic.xyz/stkbpt

### stkABPT Emissions

The Aave DAO currently emitting 360 AAVE per day to stkBPT holders. Assuming a spot price of $150/AAVE, this is equivalent to $19.71M a year.

The yield from holding stkABPT relative to the underlying, 18.21%, compares favorable to holding stkAAVE at 4.72%. Given the slashing risk is 30% for both stkAAVE and stkBPT, the delta of 13.49% represents compensation for the additional Balancer exposure.

This publication proposes reducing AAVE emissions by one-third, to 240 AAVE/day. The resulting AAVE yield on stkBPT is estimated to reduce by 4.67%, to 11.96% at current rates whilst still comparing favorably to stkAAVE at a time when swap fees and the Lido Index rate is below historical averages.

Based upon the swap size distribution routing through the pool, an overall reduction in TVL would not adversely affect users seeking to acquire AAVE. However, earning near 12% for providing liquidity in the current market is also an attractive investment option for LPs to consider.

## Specification

> This proposal when implemented via AIP will reduce the AAVE emission rate.

Reviewer: ```suggestion
This AIP reduces the AAVE emission rate to the following:
```
